workflow/kfdrc_consensus_calling.cwl

```diff
@@ -1,34 +1,266 @@
+   | kfdrc_vcf2maf_public                        | run step         | NA            | gunzip -c /sbgenomics/workspaces/0154a615-2b20-4c32-9c76-141b5eebdde1/tasks/0d982def-37a4-4dfe-8274-822cce0f5ac2/hard_filter_vcf/DELME_TEST.vcf.gz > input_file.vcf && perl /vcf2maf/vcf2maf.pl --input-vcf input_file.vcf --output-maf DELME_TEST.public.vep.maf --tumor-id BS_79SYEHY3 --normal-id BS_0KKH9VKP --ncbi-build GRCh38 --ref-fasta /sbgenomics/Projects/0154a615-2b20-4c32-9c76-141b5eebdde1/Homo_sapiens_assembly38.fasta --maf-center "." --retain-info MQ,MQ0,CAL,HotSpotAllele                                                                                                                                                                                                         |
+   | rename_protected                        | run step         | NA            | mkdir renamed/;cp /sbgenomics/workspaces/0154a615-2b20-4c32-9c76-141b5eebdde1/tasks/0d982def-37a4-4dfe-8274-822cce0f5ac2/variant_filter/DELME_TEST.filter.gatk.soft_filtered.vcf.gz renamed/DELME_TEST.protected.vcf.gz;cp /sbgenomics/workspaces/0154a615-2b20-4c32-9c76-141b5eebdde1/tasks/0d982def-37a4-4dfe-8274-822cce0f5ac2/variant_filter/DELME_TEST.filter.gatk.soft_filtered.vcf.gz.tbi renamed/DELME_TEST.protected.vcf.gz.tbi;cp /sbgenomics/workspaces/0154a615-2b20-4c32-9c76-141b5eebdde1/tasks/0d982def-37a4-4dfe-8274-822cce0f5ac2/kfdrc_vcf2maf_protected/DELME_TEST.protected.vep.maf renamed/DELME_TEST.protected.maf;                                                                                                                                                                                                         |
+   | rename_public                        | run step         | NA            | mkdir renamed/;cp /sbgenomics/workspaces/0154a615-2b20-4c32-9c76-141b5eebdde1/tasks/0d982def-37a4-4dfe-8274-822cce0f5ac2/hard_filter_vcf/DELME_TEST.vcf.gz renamed/DELME_TEST.public.vcf.gz;cp /sbgenomics/workspaces/0154a615-2b20-4c32-9c76-141b5eebdde1/tasks/0d982def-37a4-4dfe-8274-822cce0f5ac2/hard_filter_vcf/DELME_TEST.vcf.gz.tbi renamed/DELME_TEST.public.vcf.gz.tbi;cp /sbgenomics/workspaces/0154a615-2b20-4c32-9c76-141b5eebdde1/tasks/0d982def-37a4-4dfe-8274-822cce0f5ac2/kfdrc_vcf2maf_public/DELME_TEST.public.vep.maf renamed/DELME_TEST.public.maf;                                                                                                                                                                                                         |
```

Comment:
The indentation seems to be getting erratic towards the end. In all, I'm not sure how I feel about these in the doc. Seems like it will get out of control from larger, more complicated workflows.